src/main/java/team9/ddang/dog/controller/DogController.java

```diff
@@ -22,30 +24,39 @@ public class DogController {
     private final DogService dogService;
 
     @PostMapping
-    public ResponseEntity<ApiResponse<CreateDogResponse>> createDog(@RequestBody @Valid CreateDogRequest request) {
-        CreateDogResponse response = dogService.createDog(request.toServiceRequest());
-        return ResponseEntity.ok(ApiResponse.created(response)); // 생성된 강아지 정보 반환
+    public ApiResponse<CreateDogResponse> createDog(
+            @RequestBody @Valid CreateDogRequest request,
+            @AuthenticationPrincipal CustomOAuth2User customOAuth2User) {
+        Long memberId = customOAuth2User.getMember().getMemberId();
+        CreateDogResponse response = dogService.createDog(request.toServiceRequest(), memberId);
+        return ApiResponse.created(response);
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<ApiResponse<GetDogResponse>> getDogById(@PathVariable Long id) {
+    public ApiResponse<GetDogResponse> getDogById(@PathVariable Long id) {
         GetDogResponse response = dogService.getDogById(id);
-        return ResponseEntity.ok(ApiResponse.ok(response));
+        return ApiResponse.ok(response);
     }
 
     @PatchMapping("/{id}")
-    public ResponseEntity<ApiResponse<Void>> updateDog(
+    public ApiResponse<Void> updateDog(
             @PathVariable Long id,
-            @RequestBody @Valid UpdateDogRequest request
-    ) {
-        dogService.updateDog(request.toServiceRequest(id));
-        return ResponseEntity.ok(ApiResponse.ok(null)); // 성공 시 메시지만 반환
+            @RequestBody @Valid UpdateDogRequest request,
+            @AuthenticationPrincipal CustomOAuth2User customOAuth2User) {
+
+        Long memberId = customOAuth2User.getMember().getMemberId();
+        dogService.updateDog(request.toServiceRequest(id), memberId);
+        return ApiResponse.ok(null);
```

Comment:
```suggestion
        return ApiResponse.noContent();
```
이 부분 딱히 반환 값 없으시면 이렇게 하시면 될 거 예요!